quixstreams/sources/manager.py

```diff
@@ -117,6 +117,9 @@ def raise_for_error(self) -> None:
         Raise a `quixstreams.sources.manager.SourceException`
         if the child process was terminated with an exception.
         """
+        if not self.started:
+            return
```

Comment:
I think that line has no coverage, adding a test should be relatively easy, wdyt?